samples/store-locator/src/index.ts

```diff
@@ -37,9 +39,12 @@ const stores: Store[] = [];
 // Initialize and add the map
 function initMap(): void {
   distanceMatrixService = new google.maps.DistanceMatrixService();
+  originalMapTypeId = google.maps.MapTypeId.ROADMAP;
```

Comment:
why use a variable for these?

Reply:
to make it easier for developers to customize this sample for different zoomed out vs detail view map types (e.g. satellite instead of hybrid) without having to search for all instances of setting MapTypeId in the code.